patches/@mui__monorepo@6.0.0-alpha.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/packages/api-docs-builder/utils/parseTest.ts b/packages/api-docs-builder/utils/parseTest.ts
+index 14d2c70c9939cb2e71e7664c06e8f51700711772..e21e06960ffbaf05aff6e5d4d7faa123cb6105bc 100644
+--- a/packages/api-docs-builder/utils/parseTest.ts
++++ b/packages/api-docs-builder/utils/parseTest.ts
+@@ -3,7 +3,7 @@ import * as babel from '@babel/core';
+ import { readFile } from 'fs-extra';
+ import glob from 'fast-glob';
+ 
+-const workspaceRoot = path.join(__dirname, '../../../');
++const workspaceRoot = process.cwd();
```

Comment:
This is the most debatable change.
Ideally, IMHO, this should be updated in the Core repo and then the patch removed.
Without this, the `docs:api` uses `babel.config.js` from `@mui/monorepo` instead of the `mui-x` one and fails. 🙈

Reply:
Personally, I'd pass on patching `@mui/monorepo`. We own this repo, if this is the correct solution, why not make the `workspaceRoot` injectable at the source?

(I don't have the bandwidth right now to dig into the root cause of the problem though)

Reply:
> Personally, I'd pass on patching @mui/monorepo. We own this repo, if this is the correct solution, why not make the workspaceRoot injectable at the source?

Depends on the preferred timing.
I'm all for fixing the problem at the root.
I mainly wanted to unblock the migration progress.
At this point, the change can be made on monorepo side, but I believe we'll then need to do a separate PR for bumping monorepo as I think I've noticed some broken things if a more recent version of it is used. 🙈

Reply:
Ok for me if it's non-permanent

Reply:
I ran into a similar issue and fixed it at the source: https://github.com/mui/material-ui/pull/41999

Reply:
Just noting that I'll make sure that the patch is removed when we are unblocked from bumping monorepo. 😉